controllers/managedocs_controller.go

```diff
@@ -1630,21 +1635,26 @@ func (r *ManagedOCSReconciler) reconcileEgressFirewall() error {
 				return fmt.Errorf("Unable to determine machine CIDR from AWS ConfigMap")
 			}
 			// Allow egress traffic to that cidr range
-			vpcEgressRule := ovnv1.EgressFirewallRule{
-				Type: ovnv1.EgressFirewallRuleAllow,
-				To: ovnv1.EgressFirewallDestination{
-					CIDRSelector: vpcCIDR,
-				},
+			var vpcEgressRules []ovnv1.EgressFirewallRule
+			cidrList := strings.Split(vpcCIDR, ";")
+			for _, cidr := range cidrList {
+				if cidr == "" {
+					continue
+				}
+				vpcEgressRules = append(vpcEgressRules, ovnv1.EgressFirewallRule{
+					Type: ovnv1.EgressFirewallRuleAllow,
+					To: ovnv1.EgressFirewallDestination{
+						CIDRSelector: cidr,
+					},
+				})
```

Comment:
No need to use continue (a execution break) for such a simple case
```suggestion
				if cidr != "" {
					vpcEgressRules = append(vpcEgressRules, openshiftv1.EgressNetworkPolicyRule{
						Type: openshiftv1.EgressNetworkPolicyRuleAllow,
						To: openshiftv1.EgressNetworkPolicyPeer{
							CIDRSelector: cidr,
						},
					})
				}
```